### PR TITLE
added tx hash signing

### PR DESCRIPTION
**Pull request by mihaib79**
added json wallet import / export

### CODE REVIEW

File: erdgo/go.mod [hunk 1]
```diff
@@ -5,5 +5,8 @@ go 1.13
 require (
 	github.com/ElrondNetwork/elrond-go v1.1.4
```
Comment (miiu96) [inline erdgo/go.mod:6]: maybe you can update the version of elrond-go

File: erdgo/utils.go [hunk 2]
```diff
@@ -31,3 +39,29 @@ func SignTransaction(tx *Transaction, privateKey []byte) error {
 
 	return nil
 }
+
+// SignTransactionHash signs a transaction with the provided private key
+func SignTransactionHash(tx *Transaction, privateKey []byte) error {
```
Comment (miiu96) [inline erdgo/utils.go:44]: add a test also for transaction signed with hash.

File: erdgo/utils.go [hunk 3]
```diff
@@ -7,8 +7,16 @@ import (
 	"github.com/ElrondNetwork/elrond-go/crypto/signing"
 	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519"
 	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519/singlesig"
+	"github.com/ElrondNetwork/elrond-go/hashing/keccak"
 )
 
+const (
```
Comment (miiu96) [inline erdgo/utils.go:13]: maybe you can add in cmd/testTransactionSignedWithHash

File: erdgo/utils.go [hunk 4]
```diff
@@ -31,3 +39,29 @@ func SignTransaction(tx *Transaction, privateKey []byte) error {
 
 	return nil
 }
+
+// SignTransactionHash signs a transaction with the provided private key
```
Comment (miiu96) [inline erdgo/utils.go:43]: add more details about this method in comment

File: erdgo/go.mod (hunk 1 above)
Comment (mihaib79) [reply erdgo/go.mod]: Done

File: erdgo/utils.go (hunk 4 above)
Comment (mihaib79) [reply erdgo/utils.go:43]: Done

File: erdgo/utils.go (hunk 3 above)
Comment (mihaib79) [reply erdgo/utils.go:13]: Done

**Review by miiu96 — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Pull request merged**